### ISSUE TITLE
Make `simple_form` optional

### ISSUE THREAD

**Issue by ojab** [605data/strongbolt#35]
For `rails --api` apps there is no purpose of having `simple_form` dependency, probably all frontend stuff should be moved to another gem (i. e. `strongbolt-assets`).

**Comment by tobithiel** [comment]
I agree, but I'm not so sure we will have the bandwidth to tackle this soon. Is Simple Form causing errors in your Rails API app or can you use it anyways?

**Comment by ojab** [comment]
No errors, just a warning `[Simple Form] Simple Form is not configured in the application and will use the default values. Use `rails generate simple_form:install` to generate the Simple Form configuration.` on start, so no hassle at all.
